src/components/Media/MediaPopup.js

```diff
@@ -128,10 +130,8 @@ export default function MediaPopup({ open, close, triggerUpdate, mediaType, expa
           </form>
         </DialogContent>
         <DialogActions className={classes.buttons}>
-          <Button autoFocus onClick={close}>
-            Abbrechen
-          </Button>
-          <Button autoFocus onClick={handleSave} color="primary">
+          <Button onClick={close}>Abbrechen</Button>
+          <Button autofocus onSubmit={handleSave} color="primary">
```

Comment:
```suggestion
          <Button autofocus onClick={handleSave} color="primary">
```
Müssen wir doch wieder so schreiben, weil es sonst nicht funktioniert 😅